### PR TITLE
ppOpen-APPL/AMR-FDM: New package

### PR DESCRIPTION
**Pull request by tkameyama**

### CODE REVIEW

File: var/spack/repos/builtin/packages/ppopen-appl-amr-fdm/package.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+class PpopenApplAmrFdm(MakefilePackage):
+    """
+    ppOpen-APPL/AMR-FDM is an adaptive mesh refinement (AMR) framework
+    for development of 3D parallel finite-difference method (FDM)
+    applications by the following capabilities:
+
+    - Cell-based adaptive mesh structure
+    - Dynamic domain decomposition (DDD) to avoid load imbalance problem
+    - Explicit time integration
+    - Flat MPI & OpenMP/MPI hybrid parallel programming models
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/ppopen-appl-amr-fdm/package.py]: The formatting of this section will break when you run `spack info ppopen-appl-amr-fdm`. I would either remove these details, or add commas or something.

File: var/spack/repos/builtin/packages/ppopen-appl-amr-fdm/package.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    homepage = "http://ppopenhpc.cc.u-tokyo.ac.jp/ppopenhpc/"
+    url      = "file://{0}/ppohAMRFDM_0.3.0.tar.gz".format(os.getcwd())
+
+    version('0.3.0', sha256='e82217e4c949dd079a56024d3d1c1761dc8efd5ad0d26a3af83564c3db7327bb')
+
+    depends_on('mpi')
+
+    parallel = False
+    build_targets = ['default', 'install', 'advAMR3D', 'bin_install']
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/ppopen-appl-amr-fdm/package.py]: Hmm, I'm not sure I like how the "build" phase installs the package and the "install" phase adds the docs. Could we only do the building during "build" and do the rest during "install"?

Comment (alalazo) [reply var/spack/repos/builtin/packages/ppopen-appl-amr-fdm/package.py]: +1 on that

Comment (tkameyama) [reply var/spack/repos/builtin/packages/ppopen-appl-amr-fdm/package.py]: fixed.

File: var/spack/repos/builtin/packages/ppopen-appl-amr-fdm/package.py (hunk 1 above)
Comment (tkameyama) [reply var/spack/repos/builtin/packages/ppopen-appl-amr-fdm/package.py]: fixed.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**